lib/bolt/executor.rb

```diff
@@ -326,6 +326,18 @@ def wait_until(timeout, retry_interval)
       end
     end
 
+    def prompt(prompt = '', options = {})
```

Comment:
At this point, at the very least `prompt` should be required if not `options` (I would prefer it, but we don't make it required anywhere else in this class so...)